src/main/java/com/smartbear/readyapi/client/execution/TestServerApi.java

```diff
@@ -15,7 +15,7 @@
  */
 public interface TestServerApi {
 
-    ProjectResultReport postProject(File file, boolean async, HttpBasicAuth auth, @Nullable String testCaseName, @Nullable String testSuiteName, @Nullable String environment) throws ApiException;
```

Comment:
Even for API interface and implementation? I think it's enough to keep old methods in RecipeExecutor.